workalendar/asia/japan.py

```diff
@@ -20,18 +20,43 @@ class Japan(WesternCalendar, EphemMixin):
         (5, 5, "Children's Day"),
         (11, 3, "Culture Day"),
         (11, 23, "Labour Thanksgiving Day"),
-        (12, 23, "The Emperor's Birthday"),
     )
 
     def get_fixed_holidays(self, year):
         """
         Fixed holidays for Japan.
 
-        As of 2016, the "Mountain Day is being added"
+        As of 2016, the "Mountain Day" is being added
```

Comment:
nitpick: it's not necessary in the docstring, the code comments are more adapted for this kind of amendment.